src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -33,6 +39,14 @@ export const selectPriceHistoryByAssetTimeframe = createSelector(
   (priceHistory, assetId, timeframe) => priceHistory[timeframe][assetId] ?? [],
 )
 
+export const selectPriceHistoriesUnavailableByAssetsAndTimeframe = createSelector(
```

Comment:
as per my comment above, i don't think we need this selector, or it needs to contain early return logic `if (assetIds.length > 1) return false` with a corresponding comment as to why

Reply:
the equality check on line 47 is doing this already I think, although I'll admit it's difficult to read. the selector is returning true only if the number of unavailable assets is equal to the total number of assets queried, so that the balance chart is only empty if all assets are unavailable. I've changed this line to use Array.some in the latest commit which is cleaner and I renamed this selector, but see comment above. can remove if this logic is too complicated